controllers/main_controller.rb

```diff
@@ -9,3 +9,19 @@
   # combined document back to the client.
   erb :"main/welcome"
 end
+
+MyApp.get "/about" do
+  erb :"main/about"
+end
+
+MyApp.get "/favorites" do
+  erb :"main/favorites"
+end
+
+MyApp.get "/greet/Sumeet" do
```

Comment:
if I understand it correctly it can't access it because the path to those files is no longer present. If I remove them from my code suddenly nearly everything is failing.